.github/workflows/cherryPick.yml

```diff
@@ -27,6 +24,22 @@ jobs:
           token: ${{ secrets.OS_BOTIFY_TOKEN }}
           submodules: true
 
+      - name: Extract PR information
+        id: getPRInfo
+        run: |
+          echo "REPO_FULL_NAME=$(echo '${{ github.event.inputs.PULL_REQUEST_URL }}' | sed -E 's|https://github.com/([^/]+/[^/]+)/pull/.*|\1|')" >> "$GITHUB_OUTPUT"
+          echo "PR_NUMBER=$(echo '${{ github.event.inputs.PULL_REQUEST_URL }}' | sed -E 's|.*/pull/([0-9]+).*|\1|')" >> "$GITHUB_OUTPUT"
+
+      - name: Verify repository
+        run: |
+          if [ "${{ steps.GetPRInfo.outputs.REPO_FULL_NAME }}" != "Expensify/App" ]; then
+            echo "::error::❌ Cherry picks are only supported for the Expensify/App repository. Found: ${{ steps.GetPRInfo.outputs.REPO_FULL_NAME }}"
+            exit 1
+          fi
+
+      - name: Set conflict branch name
+        run: echo "CONFLICT_BRANCH_NAME=cherry-pick-staging-${{ steps.GetPRInfo.outputs.PR_NUMBER }}-${{ github.run_id }}-${{ github.run_attempt }}" >> "$GITHUB_ENV"
```

Comment:
```suggestion
        run: echo "CONFLICT_BRANCH_NAME=cherry-pick-staging-${{ steps.getPRInfo.outputs.PR_NUMBER }}-${{ github.run_id }}-${{ github.run_attempt }}" >> "$GITHUB_ENV"
```

Reply:
NAB but I think we should use `$GITHUB_OUTPUT` instead of `$GITHUB_ENV` to produce data in steps and pass it to other steps.

The reason I say that is that, even though it's a bit more robust, it's more explicit where the data is coming from, and we'll get better linting with actionlint